docs/concepts/configuration/assign-pod-node.md

```diff
@@ -142,6 +142,8 @@ If you specify multiple `nodeSelectorTerms` associated with `nodeAffinity` types
 
 If you specify multiple `matchExpressions` associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node **only if all** `matchExpressions` can be satisfied.
 
+If you remove or change the label of the node which the pod is running on,the running pod won't be stopped.It means that the affinity selection only works when the pod is scheduled. 
```

Comment:
Please replace with:

If you remove or change the label of the node where the pod is scheduled, the pod won't be removed. In other words, the affinity selection works only at the time of scheduling the pod.